server/converters/cxgtool.py

```diff
@@ -409,6 +441,17 @@ def save_metadata(container, metadata):
     with tiledb.DenseArray(a_name, mode="w") as A:
         A.meta["cxg_version"] = CXG_VERSION
         A.meta["cxg_properties"] = json.dumps(metadata)
+        if extract_colors:
+            try:
+                A.meta["cxg_category_colors"] = json.dumps(
+                    convert_anndata_category_colors_to_cxg_category_colors(adata)
```

Comment:
this (currently) can also generate a TypeError which needs to be handled somewhere